### DISCUSSION TITLE
Simple question : How to toggle IdeaVim on/off from keyboard, with a simple shortcut ?

### DISCUSSION THREAD

**Discussion by micha1805** [JetBrains/ideavim#655] [General]
The title says it all. It seems very simple but I didn't find any way to do it easily.

**Comment by AlexPl292** [comment]
Please see https://stackoverflow.com/questions/58748615/is-there-a-way-to-enable-disable-intellij-ideavim-plugin-using-keyboard-shortcut/58748985#58748985
